_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -206,6 +219,7 @@ Note that this is only support to run Foreman and Foreman Proxy themselves on EL
 * TFTP Unable to download Flatcar OS \- [#33469](https://projects.theforeman.org/issues/33469)
 
 #### Foreman - Templates
+* Come up with slightly better name for puppet7-repo parameter \- [#34088](https://projects.theforeman.org/issues/34088)
```

Comment:
I've renamed this in Redmine. I also think this is a breaking change since it lacks a DB migration. That means we should at least add an upgrade warning that we'll break provisioning setups. Noted in https://github.com/theforeman/foreman/pull/8970#issuecomment-990044976 too.

Reply:
No need, this was new parameter in 3.1

Reply:
Thanks for checking. Sounds like it could only affect users of RCs and we never give them guarantees so that's acceptable.